include/verilatedos.h

```diff
@@ -521,6 +521,25 @@ using ssize_t = uint32_t;  ///< signed size_t; returned from read()
 // Conversions
 
 namespace vlstd {
+
+template <typename T>
+struct reverse_wrapper
+{
+    const T& m_v;
+
+    explicit reverse_wrapper(const T& a_v) : m_v(a_v) {}
+    inline auto begin() -> decltype(m_v.rbegin()) { return m_v.rbegin(); }
+    inline auto end() -> decltype(m_v.rend()) { return m_v.rend(); }
+};
+
+/// \brief A iterator modifier that will iterate the passed type in reverse
+///        Usage: for(auto& value : iter::reverse(array))
```

Comment:
Add a comment explaining this is supposed to be the same as `std::ranges::reverse_view`
Note: I think the only thing that should use doxygen style these days is user facing public APIs, this isn't one.